.evergreen/run-orchestration.sh

```diff
@@ -30,13 +31,23 @@ if [ "$SSL" != "nossl" ]; then
    ORCHESTRATION_FILE="${ORCHESTRATION_FILE}-ssl"
 fi
 
+if [ ! -z "$STORAGE_ENGINE" ]; then
```

Comment:
lets throw a comment here saying the storage engine files are just that, and are not available in different topologies, auth or ssl mode, so it is intentionally overwriting the value